### DISCUSSION TITLE
Uno for visual studio 2022 preview 1.0

### DISCUSSION THREAD

**Discussion by limeniye** [unoplatform/uno#7267] [Q&A]
I'm using Visual Studio Preview 4.1 with Uno Platform, consequently I have too match problems. For example I haven't intellisense in the my xaml code and HotReload does't work.

I decided to install the same version as the UNO developers.

According to this file -> https://github.com/unoplatform/uno/blob/master/src/Uno.UI.sln, Uno is using VisualStudio preview 1.0.

I searched for such a version for 6 hours and could not find it. Can someone drop a link to it?

**Comment by limeniye** [comment]
Ok. I found
https://download.visualstudio.microsoft.com/download/pr/b6e51993-aee8-44a4-ab52-0fe6da4e98c6/ae1be4a20e0491e96397b70ae5da9f01e0c1772d9c6aa94da9bf5fc5f2c54791/vs_Professional.exe

**Reply by jeromelaban** [reply]
Uno 3.10.11 is compatible with VS 2022 Pre 4.1, within the known issues. If you created the project with VS2022, it's then using .NET 6, for which there are still issues in VS 2022 with XAML intellisense on Microsoft's side.

Which project head (target platform) did you try with?

**Reply by limeniye** [reply]
I tried with VisualStudio [1.1, 3.0, 3.1, 4.0, 4.1]. 
In all versions, I have the following problems: does not display a sentence of words, doesn't work HotReload.

**Reply by limeniye** [reply]
Third problem. When the xaml automatically closes a tag, the cursor does not go to the end of that tag, but to the very top of the page.

**Reply by limeniye** [reply]
Hot reload is working, but only in WPF Deploy*
Hot reload in the UWP doesn't work

**Reply by jeromelaban** [reply]
- XAML tag closing is a VS issue that we can't fix, [see the report here](https://developercommunity.visualstudio.com/t/xaml-editor-resets-the-cursor-position-on-element/1544841) (if you can upvote it, it will help).
- Hot Reload for UWP is still experimental on Microsoft's side in VS2022, and there may still be issues. Make sure to set the editor project selector (top left combobox) to UWP before starting the edition.

**Reply by limeniye** [reply]
- At this point, I was able to temporarily fix the closing tag issue. I downloaded Resharper.
- But the problem with the Intellisense technology still remains. I am not given a list of suggestions. Perhaps I did not configure Resharper correctly.
- But in the Xaml MAUI document all is work normally without Resharper. 
- I checked, HotReload works for me only when selected WPF-platform.

P.S. I voted
